lib/MetaCPAN/Server/Controller/Author.pm

```diff
@@ -61,9 +62,26 @@ sub get : Path('') : Args(1) {
         ['The requested field(s) could not be found'] );
 }
 
+# /author/by_user/USER_ID
 sub by_user : Path('by_user') : Args(1) {
     my ( $self, $c, $user ) = @_;
-    my $data = $self->model($c)->raw->by_user($user);
+    my $data = $self->model($c)->raw->by_user( [$user] );
+    $data or return;
+    $c->stash($data);
+}
+
+# GET:  /author/by_user?users=USER_ID1,USER_ID2...
+# POST: /author/by_user  {"users":["USER_ID1","USER_ID2"...]}
+sub by_users : Path('by_user') : Args(0) {
+    my ( $self, $c ) = @_;
+
+    # multiple user ids in 'users' param(s) (can be CSV)
+    my $users = $c->req->params->{users} // $c->req->body_data->{users};
```

Comment:
How about just using `my @users = $c->req->param('users')` here?  That should cover both `GET` and `POST` and we'd be guaranteed to get a list, so no need to check `is_arrayref` below.

https://metacpan.org/pod/Catalyst::Request#$req->param

Also, I don't think we need the split since we could send `user=foo&user=bar&user=baz` to get the same result.